cicd/build/monit-taskworker/monit_build.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-## bash ./cicd/build/monit-taskworker/monit_build.sh v3.240111
+## bash ./cicd/build/monit-taskworker/monit_build.sh
 
-TAG=v3.240325
+TAG=pypi-test2-1716833532
```

Comment:
back to this, sorry. Do we really need different containers ? IIUC you have two: one is basically the TW, the other derives from pythom:3.8 like TW.
What I worry is that we change monitor code and do not remember to go and change the hardcoded tag,
If it were not for the space in Harbor (maybe we can get more ?) I vote for building and deploying everything every time.

Reply:
we needed two different containers before the pypi times [1]. 

now that everything is in a pypi container, we can use a single container for every monit script based on the pypi TW container. and since the additions required by monit scripts are not much (it should not add much space in the additional layers), i am in favor of building the monit container every time we build the TW container.

@novicecpp opinions?

[1] generate monit required htcondor and that was readily available in the TW container, while the other scripts required pandas that i did not manage to install in the TW container.